### ISSUE TITLE
Server Crash Exploit: Combining Block of Cheese

### ISSUE THREAD

**Issue by mvroe** [ValveSoftware/Dota2-Gameplay#15991]
Today I played two ranked games in immortal draft. In both games around the 50 minute mark I tried to combine 3x Greater Healing Lotus and 1x Cheese in my inventory. 

After dropping them from my backpack to the ground, then picking them up back into my inventory (slot1 and 2) both times I immediately was disconnected from the game.
I was unable to reconnect and soon after I saw the game will not be scored and is free to leave.
Both matches did not show up in my match history and neither in DotaBuff. No MMR was lost/gained either.

I suspect that at the moment this is a reliable method to crash a match.

**Comment by CruxieTV** [comment]
i had 5-6 ranked matches the last 2 weeks where we were about to do winning push, i got dced(game crashed) and couln´t reconnect. I didn´t get MMR loss/gain but i get banned for matchmaking. First it was 5 mins then 20 and no i am at 1h+.

Guess some guys know about that or there is a different reason for it.

**Comment by 413120** [comment]
Happened in my match as well. Can easily recreate in demo lobby. Simply drop 3x greater healing lotuses (stacked, not sure if it matters), drop the cheese, and attempt to pick up the cheese and the server will crash.

**Comment by BenKiss90** [comment]
Yeah! It happened to me twice. For some reason, the components don't merge automatically, so I put them in the ground and picked them up again, forming a block of cheese. However, the game crashed.

**Comment by mvroe** [comment]
#16253 Gives a detailed explanation on how to reproduce

**Comment by MicroKiss** [comment]
Can confirm just happened the same with me, 
In a turbo match at the fountain I tried to assemble the block of cheese via dropping them to the ground and picking them up, and BOOM disconnected me and the game ended without scoring.

**Comment by 413120** [comment]
Well, this was fixed a acouple months back, but it looks like it's returned with the 7.36b update (or potentially 7.36a or 7.36 itself, but there were no reports then.)

**Comment by ksarlary** [comment]
It happened today again, maybe some of updates broke the fix or something, but 60+ mins game gone into nowhere

**Comment by bwvalve** [comment]
This should have been fixed as of yesterday. Please comment with your accountID if this happens again
